build.sbt

```diff
@@ -1,8 +1,7 @@
 import play.core.PlayVersion.{ current => playVersion }
 import interplay.ScalaVersions._
 
-val scala211    = "2.11.12"
-val AkkaVersion = "2.5.26"
+val AkkaVersion = "2.5.23"
```

Comment:
I think we can go Akka 2.6 here. :-)

Reply:
Tried that -- there are several tests that fail in 2.6.0 and I'm not sure why.  I'd rather do the akka upgrade in a different PR anyway.